ha/monitor/k8s/parser.py

```diff
@@ -22,13 +22,20 @@
 from ha.monitor.k8s.const import K8SEventsConst
 from ha.monitor.k8s.const import AlertStates
 from ha.monitor.k8s.const import EventStates
-from ha.monitor.k8s.alert import K8sAlert
 
 from cortx.utils.log import Log
+from ha.fault_tolerance.event import Event
+from ha.fault_tolerance.const import HEALTH_ATTRIBUTES, HEALTH_EVENT_SOURCES
+
 
 class ObjectParser:
     def __init__(self):
-        pass
+        self.payload = {HEALTH_ATTRIBUTES.SOURCE.value: HEALTH_EVENT_SOURCES.MONITOR.value, \
```

Comment:
No..This is not needed in utils. I am forming this payload dictionary here to pass it to utils's set_payload() method and then it will return full event in the defined format.

Reply:
okay